Gems/WarehouseSample/gem.json

```diff
@@ -0,0 +1,25 @@
+{
+    "gem_name": "WarehouseSample",
+    "display_name": "WarehouseSample",
+    "license": "Apache-2.0 or MIT",
+    "license_url": "https://opensource.org/licenses/Apache-2.0",
+    "origin": "WarehouseSample",
+    "origin_url": "https://github.com/RobotecAI/WarehouseSample/tree/main/Gems/WarehouseSample",
```

Comment:
should this be updated to the new location?

Reply:
Done.